include/veriblock/state_manager.hpp

```diff
@@ -114,14 +57,14 @@ class StateManager {
   RepositoryManager database;
 
  public:
-  StateManager(const std::string& name) : database(name) { database.open(); }
+  StateManager(const std::string& name, std::string path = "")
+      : database(path + "\\" + name) {
```

Comment:
Why this change? We could add separators outside, by the StateManager users.

Reply:
fixed